packages/components/src/components/accordion/_accordion.scss

```diff
@@ -23,6 +23,22 @@
     width: 100%;
   }
 
+  .#{$prefix}--accordion--start {
+    .#{$prefix}--accordion__heading {
```

Comment:
Could you un-nest selectors here?

Reply:
Thanks @joshblack! Just pushed an update that unnests these selectors 👍